pegasus/router.rb

```diff
@@ -179,8 +179,8 @@ def self.set_max_age(type, default)
 
   get '/style.css' do
     content_type :css
-    css, css_last_modified = combine_css 'styles', 'styles_min'
-    last_modified(css_last_modified) if css_last_modified > Time.at(0)
+    css, digest = combine_css 'styles', 'styles_min'
+    etag digest
```

Comment:
so now we cache the digested version, correct? this looks good to me

Reply:
We serve the same content as before, we are just telling all the caching layers to use the digest in the response header as the differentiator for new content, rather than the last-modified date.